homeassistant/components/tibber/sensor.py

```diff
@@ -235,3 +236,8 @@ def unit_of_measurement(self):
     def unique_id(self):
         """Return a unique ID."""
         return f"{self.device_id}_rt_consumption"
+
+    @property
+    def device_class(self):
+        """Return the device class of the sensor."""
+        return DEVICE_CLASS_POWER
```

Comment:
This PR should not have been merged.

Either set a device class or set an icon. Not both.
Currently, this sensor sets both.

Reply:
I will remove the ICON

Reply:
https://github.com/home-assistant/core/pull/41604